homeassistant/components/motion_blinds/config_flow.py

```diff
@@ -31,33 +37,71 @@ def __init__(self):
         """Initialize the Motion Blinds flow."""
         self.host = None
         self.key = None
+        self.ips = []
```

Comment:
The home assistant style I typically see is to prefix private members with underscores, so might be worth updating in a followup.

Reply:
you are right, the _ means it is a private variable.
Will make a commit

Reply:
done